src/modules/comments.js

```diff
@@ -0,0 +1,25 @@
+const baseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/LOf5L284gEd1pGlHG0AY/comments';
+
+const postShowCommentData = async (userName, comment, id) => {
+  const response = await fetch(baseUrl, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      username: userName,
+      comment,
+      item_id: id,
+    }),
+  });
+  return response;
+};
```

Comment:
[OPTIONAL] This function does not have any error handling ❌, which means if there is an issue with the API call, the function will fail silently 🤫 . You can add a try-catch block to handle errors and provide useful feedback to the user. 🤓 

Make sure to address the same issue in the whole codebase to provide a better experience 😉